_pages/install/migration/calydon.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Calydon Theme
```

Comment:
Should this be "Deprecation of Calydon Theme"?

Reply:
@gladius-mtl the above is just a suggestion, although text that follows make it clear. I'm approving this PR as I don't have anything else to add. But since @tobi-or-not-tobi is the creator of the theme structure his review will be the final say.

Reply:
Sounds very good! Thank you very much 🙂

Reply:
Done 🙂